tests/multi_objective_tests/visualization_tests/test_pareto_front.py

```diff
@@ -4,10 +4,11 @@
 from optuna.multi_objective.visualization import plot_pareto_front
 
 
-def test_plot_pareto_front_2d() -> None:
+@pytest.mark.parametrize("enable_include_dominated_trials", [False, True])
```

Comment:
A nit but I think that the "enable_" prefix can be removed from the variable name.